lib/audited/auditor.rb

```diff
@@ -141,15 +139,11 @@ def audited_attributes
       protected
 
       def revision_with(attributes)
+        instance_variables = { "new_record" => self.destroyed?, "persisted" => !self.destroyed?, "readonly" => false, "destroyed" => false, "_destroyed" => false, "marked_for_destruction" => false }
```

Comment:
Do you still use audited? You're certainly welcome to touch up this PR. We welcome your input. :)